test/parallel/test-timers-unrefd-interval-still-fires.js

```diff
@@ -5,23 +5,20 @@
 const common = require('../common');
 
 const TEST_DURATION = common.platformTimeout(1000);
-const N = 3;
-let nbIntervalFired = 0;
+let N = 3;
 
-const keepOpen = setTimeout(() => {
-  console.error('[FAIL] Interval fired %d/%d times.', nbIntervalFired, N);
-  throw new Error('Test timed out. keepOpen was not canceled.');
-}, TEST_DURATION);
+const keepOpen =
```

Comment:
nit: This is an uncommon wrapping, IMHO
```js
const keepOpen = setTimeout(
  common.mustNotCall('Test timed out. keepOpen was not canceled.'),
  TEST_DURATION
);
```
is more "regular".

Alltought
```js
const keepOpenMsg = 'Test timed out. keepOpen was not canceled.'
const keepOpen = setTimeout(common.mustNotCall(keepOpenMsg), TEST_DURATION);
```
is usually my preference.